### PR TITLE
Re-enable FIWARE support

### PR DESCRIPTION
**Pull request by lcbm**
(Thanks for sending a pull request! Please fill out the blanks below.)

What does this implement/fix? Explain your changes.
---------------------------------------------------

This pull request aims to re-enable FIWARE cloud support. Also, it updates the `package-lock.json` in a previous commit. 

Does this close any currently open issues?
------------------------------------------
(Use the special github keywords to reference issues: https://help.github.com/en/articles/closing-issues-using-keywords)

N/A :smiley: 

Any relevant logs, error output, etc?
-------------------------------------
(If it’s long, please paste to https://pastebin.com/ and insert the link here.)

N/A :smiley: 

Any other comments?
-------------------

Nope :smiley: 

Where has this been tested?
---------------------------
(Describe your environment setup here.)

**Operating System/Platform:** Linux 5.4.6-arch1-1 x86_64 GNU/Linux
**NPM Version:** v13.5.0 (via [nvm](https://github.com/nvm-sh/nvm) v0.35.2)
**NodeJS Version:** 6.13.4  (via [nvm](https://github.com/nvm-sh/nvm)  v0.35.2)

(Add more information here if needed.)

This commit reverts 757e9cf79cb90d088fc1507ebb2d1286a73afa29.

### CODE REVIEW

**Comment by lcbm** [comment]
This **draft PR** will be **ready for review** when [knot-fog-connector-fiware's v3.0.0 bump PR](https://github.com/CESARBR/knot-fog-connector-fiware/pull/18) is approved, so that `package-lock.json` can be updated with the published npm package info.

**Comment by ramonhpr** [comment]
Since the PR https://github.com/CESARBR/knot-fog-connector-fiware/pull/18 is approved I'm removing this from draft.

**Review by ramonhpr — CHANGES_REQUESTED** [review]
About commit description: This commit/patch reverts <hash>

**Comment by lcbm** [comment]
@ramonhpr 

> About commit description: This commit/patch reverts

Before committing, I was actually in between what you suggested and as is :stuck_out_tongue: 

I opted to leave it as is because it seems that it's a pretty standard convention across repositories (e.g.: [angular](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-angular#revert) and [electron](https://github.com/electron/electron/blob/eb2d2264d02abd7f34df36809d44752aaf93da93/patches/v8/revert_cleanup_switch_offset_of_to_offsetof_where_possible.patch)).

I can change it, though. Just let me know what you think! :smiley:

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "start:debug": "babel-node debug src"
   },
   "dependencies": {
+    "@cesarbr/knot-fog-connector-fiware": "2.0.0",
```
Comment (netoax) [inline package.json]: 3.0.0

**Review by netoax — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (lcbm) [reply package.json]: Updated! :smile:

**Review by ramonhpr — APPROVED** [review]

**Review by netoax — APPROVED** [review]

**Pull request merged**